site/kubernetes.md

```diff
@@ -0,0 +1,97 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
+
+This is especially useful when making configuration changes to the domain that also need to be reflected in the domain resource file. For example, adding a cluster to the domain only requires that it be added to the `topology` section of the WDT model, then a new domain resource file can be generated to apply to Kubernetes.
+
+More information about the Oracle WebLogic Server Kubernetes Operator can be found [here](https://oracle.github.io/weblogic-kubernetes-operator).
+
+NOTE: The Extract Domain Resource Tool is available with WDT releases 1.7.0 and later.
+
+Here is an example command line for the Extract Domain Resource Tool:
+```
+<wls-deploy-home>/bin/extractDomainResource.sh -oracle_home /tmp/oracle	-domain_home /u01/mydomain -model_file /tmp/mymodel.yaml -variable_file /tmp/my.properties -domain_resource_file /tmp/operator/domain-resource.yaml
```

Comment:
Formatting in this line there are several blank spaces between /tmp/oracle and -domain_home